src/common/lint/add-linters.ts

```diff
@@ -82,4 +85,5 @@ export const addLinters = (props: AddLintersProps): void => {
 
   const finalConfig = deepMerge([{}, ...allConfigs, mergedArrayProperties])
   new JsonFile(project, '.eslintrc.json', {obj: finalConfig, marker: false})
+  new JsonFile(project, 'cspell.json', {obj: cSpellConfig})
```

Comment:
@quesabe are we shipping static file? There are no methods to modify it?

Reply:
Yes, currently it is a static JSON file. There are ways to modify it as a regular JSON within projen and no special wrapper around it. BTW there is an example of a modification within this repo (in `.projenrc.ts`.

I did not try to make a wrapping component because I'm not sure of the API that teams might need. And form the experience with CodegenConfig it might just be left unused with people inventing their own hacks.

For now we have a starting point that is working. So let's discuss the API for it.
I see two distinct approaches:
1. Provide a wrapper with custom methods to modify particular properties.
2. Let a user input his own config (of type`CSpellConfig`) and merge it with the default one (or maybe just replace it).

Reply:
@quesabe I would express my general attitude about projen here – if we can build a limited API which covers most of the devs needs, and thus makes manual editing obsolete then it's the right way.

Merging this PR, let's discuss in Slack.